docker-compose.yml

```diff
@@ -1,39 +1,59 @@
-version: '3.4'
+## Updated to work with the latest version of 
+## Foreman. Implemented fix for missing tables
+## in the database by running rake. 
+
+## To set the default admin user password run the
+## following after running: docker compose up -d
+## and waiting for all containers to become healthy
+##
+## docker exec container-name-here bundle exec bin/rake permissions:reset username=admin password=admin
+
+version: '3.5'
+name: foreman
+
 services:
   db:
+    image: postgres:17-alpine
+    restart: unless-stopped
+    hostname: db.example.com
     environment:
-      - POSTGRES_USER=foreman
       - POSTGRES_PASSWORD=foreman
-      - POSTGRES_DATABASE=foreman
-      - PGDATA=/var/lib/postgresql/data/pgdata
-    hostname: db.example.com
-    image: postgres:12
+      - PGDATA=/var/lib/postgresql/data
     ports:
       - '5432'
-    restart: always
+    networks:
+      - foreman_net
     healthcheck:
-      test: ["CMD-SHELL", "nc -z 127.0.0.1 5432 || exit 1"]
-      interval: 30s
-      timeout: 30s
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 10s
+      timeout: 5s
       retries: 3
     volumes:
       - db:/var/lib/postgresql/data
 
   app: &app_base
     image: quay.io/foreman/foreman:develop
-    command: bundle exec bin/rails server -b 0.0.0.0
+    networks:
+      - foreman_net
+    command: 
+      - /bin/bash
+      - -c
+      - |
+        bundle exec bin/rake db:reset
```

Comment:
Doesn't this wipe the DB on every restart? I don't like that if I'm honest. Too easy to lose work.